### ISSUE TITLE
appplet icon size on hidpi screens

### ISSUE THREAD

**Issue by MaxJonasG** [budgie-desktop/budgie-desktop#793]
**Operating System**: Ubuntu 16.04 LTS
**Budgie version**: i.e. Budgie 10.2.9-2

**The issue encountered**
I hope this is the right place to log this issue, I'm new to Budgie and Github.

While the Menu Button scales with the Panel the other Applet icons don't. This is particularly annoying on HiDPI screens (e.g. Dell XPS13) as the icons therefore are really small and hard to get. 
![screenshot from 2017-02-05 13-57-56](https://cloud.githubusercontent.com/assets/25565941/22626764/15d3d402-ebad-11e6-8561-78390337b025.png)
this also true for icons on windows but they are spaced further apart.

**Comment by wezm** [comment]
Yep I ran into this on my HiDPI screen too. I poked around in the code for a bit trying to make it use larger icons but did not succeed. From reading the code my understanding is budgie uses the window icon, which it appears many applications don't make a large version available. Seems it would be better to use the application icon from the desktop file... but that's largely speculation on my part.

**Comment by ikeydoherty** [comment]
We use an application icon where it is available, actually. If the application then overrides it later with an icon, we have to do what they tell us

**Comment by MaxJonasG** [comment]
Sorry, I think we are talking about different things here, I mean the icons on the right (Wifi, Battery etc), not the icons of running applications on the left.

**Comment by ikeydoherty** [comment]
Oh. Well wifi applet isn't part of Budgie, so that's dismissed. Battery is, so we'll see why its not resizing.

**Comment by wezm** [comment]
> If the application then overrides it later with an icon, we have to do what they tell us

They all look mostly right using `gnome-shell` and the Dash to Panel extension. Either their aren't doing it the correct way or there is another mechanism?

![image](https://cloud.githubusercontent.com/assets/21787/22998222/e6ab2164-f429-11e6-853d-e4c56e997a2b.png)

**Comment by kmahyyg** [comment]
Same questions, still have no idea. Maybe we could follow gnome-shell method: just modify css of gnome shell?
